pam/integration-tests/vhs-helpers_test.go

```diff
@@ -113,6 +113,9 @@ var (
 	// number of values of the same content we want to match.
 	vhsWaitNth = regexp.MustCompile(`\bWait\+Nth\((\d+)\)(@\S+)?[\t ]+(/(.*)/|(.*))`)
 
+	// vhsTypeAndWaitCLIPassword adds support for typing the CLI password, waiting for the expected output.
+	vhsTypeAndWaitCLIPassword = regexp.MustCompile(`(.*)\bTypeCLIPassword[\t ]+"((?:[^"\\]|\\.)*)"`)
```

Comment:
the variable name implies that the command is `TypeAndWaitCLIPassword`, but it's `TypeCLIPassword`

Reply:
Yeah, true... But it looked quite verbose to me... Since at the end it looked quite clear what we're doing anyways, but let me know if you prefer to mention it anyways.

Reply:
Ok fine with me